docs/docs-content/vertex/install-palette-vertex/vertex-management-appliance.md

```diff
@@ -20,6 +20,10 @@ to boot the nodes, which are then clustered to form a Palette management cluster
 Once Palette VerteX has been installed, you can download pack bundles to create your cluster profiles. You will then be
 able to deploy clusters in your environment.
 
+There is an additional option to download and install the Third Party pack that contains complementary functionality to
+Palette VerteX. This pack is not required for Palette VerteX to function, but it does provide additional features and
+capabilities that enhance the Palette VerteX experience.
```

Comment:
Like with the Palette comment @Eulerizeit @tiwarisumit3 

I have kept this fairly generic, but feel free to suggest any specific wording regarding the third-party pack functionality.

Reply:
The relevant information is now included with links to relevant doc pages.